wsgi/odyssy/events/models.py

```diff
@@ -9,13 +9,13 @@
 
 class Event(models.Model):
     name = models.CharField(max_length=30, default="Sample Name")
-    start_time = models.DateTimeField(default=datetime.datetime.now())
-    end_time = models.DateTimeField(default=datetime.datetime.now())
+    start_time = models.DateTimeField(default=datetime.datetime.now)
+    end_time = models.DateTimeField(default=datetime.datetime.now)
     description = models.TextField(max_length=300)
     place = models.CharField(max_length=50, null=True, default=None, blank=True)
 
     def __str__(self):
-        event_str = self.name  + " on " + str(self.start_time.day) + \
+        event_str = self.name + " on " + str(self.start_time.day) + \
             " - " + calendar.month_abbr[self.start_time.month] + \
             " - " + str(self.start_time.year)
         if self.place is not None:
```

Comment:
What if administrator accidentally assigned place as  `" "`. `self.place` will return `True` in this case, but it is invalid place. So, every time before inserting a model object, we will have to strip the place. In that case, our database object is going to be changed.  Moreover, `clean()` is called before a model object is inserted into the database, so we will be making changes in model object before it's insertion.

I think it's better when `if self.place` and `if self.place is not None` returns same result. So, shall I change it @singhpratyush Sir? Also, it is mandatory to keep a check for strings with whitespace(s).